src/TestFSharp/TestFSharp.fsproj

```diff
@@ -0,0 +1,85 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <SchemaVersion>2.0</SchemaVersion>
+    <ProjectGuid>ca8dab68-17e9-48bf-afa3-9ccfd4b0a3cd</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <RootNamespace>TestFSharpGrains</RootNamespace>
+    <AssemblyName>TestFSharpGrains</AssemblyName>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFSharpCoreVersion>4.4.0.0</TargetFSharpCoreVersion>
```

Comment:
Thanks for that link @davidpodhola  Wow -- .NET versioning and dependencies have now got so complex we are soon going to need the "shading" functionality from Maven!

I think the high-level point here is that the dependency should be on FSharp.Core **v4.x** rather than the v4.**4** version in VS2015. 
We still need to be able to compile on VS2013 (v4.**3**) and AFAICT none of the F# functionality being tested is specific to v4.4 (VS2015)